fixtures/bare-return.go

```diff
@@ -31,3 +31,38 @@ func bare4(a string, b int) string {
 func bare5(a string, b int) {
 	return
 }
+
+// NR tests for issue #280
+func f280_1() (err error) {
+	func() {
+		return
```

Comment:
I'm missing something here. Why we don't report an error?

Reply:
A _bare return_ is a `return` that makes _implicit_ reference to the _named_ return values of the function 
The scope of the `return` at line 38 is the function literal. That func literal does not have return values thus the `return` it is not a bare return